.github/workflows/subgraph.yml

```diff
@@ -0,0 +1,35 @@
+name: Subgraph
+
+on:
+  pull_request:
+    branches:    
+      - main
```

Comment:
Should we start with `develop`?

Reply:
We discussed it in the call and agreed with `main` if I remember correctly. The ABI path is going to point to the Mumbai deployments, and not deploying a new version for every subgraph change that are in PRs made to `develop` will cause problems.
I don't see an advantage to adding `develop` here; Especially since Subgraph changes should be one of the last things to be done before a new release.